include/uapi/sound/sof/ext_manifest.h

```diff
@@ -27,9 +27,9 @@
 
 /* Build u32 number in format MMmmmppp */
 #define SOF_EXT_MAN_BUILD_VERSION(MAJOR, MINOR, PATH) ((uint32_t)( \
-	((MAJOR) << 24) | \
-	((MINOR) << 12) | \
-	(PATH)))
+	(UL(MAJOR) << 24) | \
+	(UL(MINOR) << 12) | \
+	UL(PATH)))
```

Comment:
@ktrzcinx TBH I don't understand what this is trying to achieve. And I don't understand the commit message.  "only positive values should be used" - so is this supposed to change behaviour for when the macro is wrongly used with negative numbers? I just tried and built a test program with both those definitions printing the result of `SOF_EXT_MAN_BUILD_VERSION(-1, -2, -3)` and there were no warnings for either of them and the result was the same. I just used `-Wall` so maybe with some more detailed warning flags it would make a difference, but I hardly can imagine that. Basically what that precompiles to is `((-1UL) << 24 | (-2UL) << 12 | (-3UL))` which should seem perfectly valid from the compiler's PoV. Could you please explain the purpose?

Reply:
Actually I thought about big values, like `SOF_EXT_MAN_BUILD_VERSION(1 << 7, 2, 3)` but I forget about casting built number to `uint32_t`, so this change actually change nothing. I remove this commit.